content/03.results.md

```diff
@@ -224,46 +235,47 @@ The mutual exclusivity of signatures 3 and MMR corroborates a previous report su
\ No newline at end of file
+Indeed, we detected significant (Bonferroni-corrected p < 0.05) upregulation of _MYC_ and _E2F_ targets, as well as G2M (cell cycle phase following S phase) in both embryonal and diffuse astrocytic and oligodendroglial tumors compared to several other cancer groups.
+In contrast, low-grade astrocytic tumors showed significant downregulation (Bonferroni-corrected p < 0.05) of these pathways.
+Schwannomas and neurofibromas, which have a documented inflammatory immune microenvironment of T and B lymphocytes as well as tumor-associated macrophages (TAMs), are driven by upregulation of cytokines such as IFN$\gamma$, IL-1, and IL-6, and TNF$\alpha$ [@doi:10.1093/noajnl/vdaa023].
+We observed this trend in GSVA hallmark pathways (Bonferroni-corrected p < 0.05) (**Figure {@fig:Fig5}B**) and immune cell deconvolution, which showed that monocytes dominated the identified immune cells in these tumors (**Figure {@fig:Fig5}C**).
+We also observed significant upregulation of pro-inflammatory cytokines IFN$\alpha$ and IFN$\gamma$ in LGG and craniopharyngiomas compared to medulloblastoma and ependymoma tumors (Bonferroni-corrected p < 0.05), both of which showed significant down-regulation of these cytokines (**Figure {@fig:Fig5}B**).
+Together, these results supported previous proteogenomic findings of lower immune infiltration in aggressive medulloblastomas and ependymomas versus higher immune infiltration in _BRAF_-driven LGG and craniopharyngiomas [@doi:10.1016/j.cell.2020.10.044]. 
+
+
+Although CD8+ T-cell infiltration across all cancer groups was quite low (**Figure {@fig:Fig5}C**), we observed some signal in specific cancer molecular subtypes (medulloblastoma Groups 3 and 4) as well as outlier tumors (BRAF-driven LGG, BRAF-driven and wildtype ganglioglioma, H3 wildtype HGG; **Figure {@fig:S6}E**) 
+Surprisingly, the classically immunologically-cold HGG and DMG tumors [@doi:10.1186/s40478-018-0553-x; 10.1093/brain/awab155] contained higher overall fractions of immune cells, with monocytes, dendritic cells, and NK cells the most prevalent (**Figure {@fig:Fig5}C**).
+This finding suggests that quanTIseq might actually be capturing microglia within these immune cell fractions.
```

Comment:
```suggestion
Thus, we suspect that quanTIseq might actually be capturing microglia within these immune cell fractions.
```

try to mix it up with not saying "this finding" or similar so many times and this brings it back to a present tense, what we think/hypothesize vs a plain statement of results